services/symmetric/README.md

```diff
@@ -0,0 +1,16 @@
+#### Setup cloud instance
+
+```
+docker-compose run  cloudSymmetric ./dbimport --engine cloud-f7e41e48-ec79-4c78-9db6-37c0c4f78326 --format XML --alter-case ../samples/create-sample.xml
```

Comment:
I think `create-sample.xml` is missing from the repo

Reply:
Added and reworded it.